nmrfx-processor/src/test/python/testoperations.py

```diff
@@ -975,5 +975,10 @@ def testZFInverse(self):
         self.assert_(size == size3)
 
 
+#if __name__ == '__main__':
+#    unittest.main()
+
```

Comment:
I think the commented out lines should be deleted if they aren't needed anymore
```suggestion
```

Reply:
Lines deleted